portal/static/js/src/profile.js

```diff
@@ -1910,22 +1910,25 @@ export default (function() {
                 self.initUserRoles();
             },
             initAuditLogSection: function() {
-                var self = this;
+                var self = this, errorMessage = "";
                 $("#profileAuditLogTable").html(Utility.getLoaderHTML());
                 $("#profileAuditLogTable .loading-message-indicator").show();
                 this.modules.tnthAjax.auditLog(this.subjectId, {useWorker:true}, function(data) {
                     if (data.error) {
-                        $("#profileAuditLogTable").html("");
-                        $("#profileAuditLogErrorMessage").text(i18next.t("Problem retrieving audit log from server."));
-                        return false;
+                        errorMessage = i18next.t("Problem retrieving audit log from server.");
                     }
                     if (!data.audits || data.audits.length === 0) {
+                        errorMessage = i18next.t("No audit log item found.");
```

Comment:
I think this is an error message we can reuse...
```suggestion
                        errorMessage = i18next.t("no data returned");
```